src/main/java/mchorse/metamorph/api/MorphManager.java

```diff
@@ -242,15 +242,7 @@ public AbstractMorph morphFromNBT(NBTTagCompound tag)
      */
     public void applySettings(AbstractMorph morph)
     {
-        if (morph.settings != MorphSettings.DEFAULT)
-        {
-            return;
-        }
-
-        if (this.activeSettings.containsKey(morph.name))
-        {
-            morph.settings = this.activeSettings.get(morph.name);
-        }
+        morph.setActiveSettings(this.activeSettings.get(morph.name));
```

Comment:
Note here that `MorphManager` only plays part of the role in initializing morph settings. The morph objects themselves handle the bulk of the state handling (via `EntityMorph.initializeSettings` or `AbstractMorph.initializeSettings`)